kalite/control_panel/static/js/control_panel/bundle_modules/data_export.js

```diff
@@ -0,0 +1,9 @@
+var $ = require("base/jQuery");
+var DataExportView = require("data_export/views").DataExportView;
+var DataExportStateModel = require("data_export/models").DataExportStateModel;
+
+module.exports = {
+	$: $,
```

Comment:
It gets imported namespaced under the module. Just like if you did:
```python
import data_export
data_export.$("this")
```
in Python.